source/_integrations/atag.markdown

```diff
@@ -0,0 +1,188 @@
+---
+title: Atag
+description: Instructions on how to setup Atag integration.
+logo: atag.png
+ha_category:
+  - Climate
+  - Water heater
+  - Sensor
+ha_release: 0.105
+ha_iot_class: Local Polling
+---
+
+The `Atag` integration allows Home Assistant to connect to [Atag One](https://atag-one.com) thermostats, reporting and setting its status.
+The integration implements the following platforms.
+
+- Climate
+- Water Heater
+- Sensor
+
+## Configuration
+
+The Atag integration can be enabled directly from the config flow UI.
+In a regular setup only the IP address has to be provided (UDP discovery planned for future release).
+
+Alternatively the integration can be setup through configuration.yaml.
+This way additional (non-default) sensors can be enabled. Only the `host` field is required.
+
+```yaml
+# Minimal setup through configuration.yaml
+atag:
+  host: YOUR_ATAG_ADDRESS
+```
+
+{% configuration %}
+host:
+  description: Atag host or ip address.
+  required: true
+  type: string
+email:
+  description: Email registered in the Atag App.
+  required: false
+  type: string
+port:
+  description: Port to reach the Atag Api. Only needed if connecting through alternative routes.
+  required: false
+  type: integer
+scan_interval:
+  description: Polling frequency in seconds. Defaults to 30, increasing may overload the device.
+  required:  false
+  type: integer
+sensors:
+  description: List of sensors to add. Uses default list if not specified. See below for the full list.
```

Comment:
We used to call this "MONITORED_CONDITIONS", which we try to remove/avoid nowadays. Make sure your integration adhere to: <https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md>

If you want to prevent all sensor showing up when initially setting up the integration, Home Assistant offers a way to disable non-popular sensors by default, allowing the user to enable them if they wish. See: <https://developers.home-assistant.io/docs/en/next/entity_registry_disabled_by.html#integrations-setting-default-value-of-disabled_by-for-new-entity-registry-entries>

Reply:
OK - this will require some changes in the way the component is currently set up. Will make the necessary changes in the coming days